src/utils/methods.js

```diff
@@ -259,7 +259,7 @@ export const parseSpeakerAuditLog = (logString) => {
     const logEntries = logString.split('|');
     const userChanges = {};
     for (const entry of logEntries) {
-        const emailMatch = entry.match(/\b[\w.-]+@[\w.-]+\.\w{2,}\b/);
+        const emailMatch = entry.match(/[\w.+-]+@[\w.-]+\.\w{2,}\b/);
```

Comment:
i do think this could be a problem
matching email by regex is not so a trivial affair
please check this 
http://www.ex-parrot.com/~pdw/Mail-RFC822-Address.html
https://emailregex.com/
https://www.regular-expressions.info/email.html